web/src/pages/contact/index.js

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import Layout from '../../components/Layout/Layout';
+import { Container, Row, Col, CarouselItem } from 'react-bootstrap';
+import Title from '../../components/UI/Title/Title';
+
+const ContactPage = ({ data }) => {
+
+  return (
+    <Layout pageTitle="Contact Us | Awesome Inc">
+    <Container>
+        <Title className="text-center my-5">CONTACT US</Title>
```

Comment:
Let's use `text-uppercase` here

Reply:
I've made the change. Let me know if there's anything else.